apps/site/models/permissions.py

```diff
@@ -134,6 +134,16 @@ def to_dict(self):
     def __unicode__(self):
         return self.user.username
 
+    # TODO: This seems a little dicey if Im not inheriting from BasePermissions?
```

Comment:
I think this is good -- it's relying on the project's can_view/edit/manage methods, which do rely on BasePermissions.